setup.py

```diff
@@ -0,0 +1,12 @@
+from distutils.core import setup
+
+setup(
+    name='nerdouts',
+    version='0.1dev',
```

Comment:
FWIW [PEP 440](https://www.python.org/dev/peps/pep-0440/#developmental-releases) specifies `X.Y.devN` for version

Reply:
lol, good catch